editor/sidebar/post-taxonomies/tags-selector.js

```diff
@@ -1,36 +1,134 @@
 /**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { unescape, find } from 'lodash';
+
+/**
  * WordPress dependencies
  */
 import { Component } from 'element';
-import FormTokenField from 'components/form-token-field';
+import { Spinner, FormTokenField } from 'components';
+import { getEditedPostAttribute } from '../../selectors';
+import { editPost } from '../../actions';
+
+const DEFAULT_TAGS_QUERY = {
+	number: 1000,
+	order_by: 'count',
```

Comment:
The correct argument name is `orderby`:

https://developer.wordpress.org/rest-api/reference/tags/#list-tags